src/main/java/ru/sbtqa/tag/parsers/TestPullRequest.java

```diff
@@ -0,0 +1,13 @@
+package ru.sbtqa.tag.parsers;
+
+public class TestPullRequest {
+    
+    public void thrower() {
+        try {
+            int i = 19;
+            System.out.println(i);
+        } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)